index.html

```diff
@@ -0,0 +1,183 @@
+                    <h4>Multi-Post Stories Gain+Glory</h4>
+                    <ul class="button-detail"> 
+                        <li>Ruby on rails</li>
+                        <li>css</li>
+                        <li>JavScript</li>
+                        <li>html</li>
+                    </ul>
+                    <button class="see-project">See project </button>
+                </div> 
+            </div>
+            <div class="project3">  
+                 <div class="project-detail">
+                    <h4>Multi-Post Stories Gain+Glory</h4>
+                    <ul class="button-detail"> 
+                        <li>Ruby on rails</li>
+                         <li>css</li>
+                        <li>JavScript</li>
+                        <li>html</li>
+                    </ul>
+                     <button class="see-project">See project </button>
+                 </div> 
+            </div>
+            <div class="project4">  
+                 <div class="project-detail">
+                    <h4>Multi-Post Stories Gain+Glory</h4>
+                    <ul class="button-detail"> 
+                        <li>Ruby on rails</li>
+                        <li>css</li>
+                        <li>JavScript</li>
+                        <li>html</li>
+                    </ul>
+                    <button class="see-project">See project </button>
+                </div> 
+            </div>
+             <div class="project5">  
+                <div class="project-detail">
+                    <h4>Multi-Post Stories Gain+Glory</h4>
+                    <ul class="button-detail"> 
+                        <li>Ruby on rails</li>
+                        <li>css</li>
+                        <li>JavScript</li>
+                        <li>html</li>
+                    </ul>
+                    <button class="see-project"> <a href="https://github.com/hiwina"></a> See project </button>
+                </div> 
+            </div>
+            <div class="project6">  
+                <div class="project-detail">
+                    <h4>Multi-Post Stories Gain+Glory</h4>
+                    <ul class="button-detail"> 
+                        <li>Ruby on rails</li>
+                        <li>css</li>
+                        <li>JavScript</li>
+                        <li>html</li>
+                    </ul>
+                    <button class="see-project">See project </button>
+                </div> 
+            </div>
+        </section>
+    </section>
+    <section class="about back-image-2">
+        <div class="about-primary-text">
+            <div class="back-image-3"></div>
+                <h3>About me</h3>
+            </div>
+        <div class="about-supporting-text">
+            I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.
+        </div >
```

Comment:
- [ ] Divs are so convenient to use that you might not realize when you are misusing them. For semantic reasons, this would be better if wrapped around a p tag instead of a div. Kindly make this change.